### PR TITLE
Added extensions to drakcore and drakpush.

### PR DESCRIPTION
**Pull request by kscieslinski**
Drakrun expects that the extension is passed in task header. I've also moved `timeout`, `start_command` and `override_uid` from task_resources to task_payload as this is what drakrun expects.

### CODE REVIEW

File: drakrun/drakrun/main.py [hunk 1]
```diff
@@ -297,7 +297,7 @@ def process(self):
             extension = 'dll'
         self.log.info("Running file as %s", extension)
 
-        file_name = self.current_task.payload.get("file_name", f"malwar.{extension}")
+        file_name = self.current_task.payload.get(f"file_name.{extension}", f"malwar.{extension}")
```
Comment (icedevml) [inline drakrun/drakrun/main.py:300]: :thinking:

File: drakrun/drakrun/drakpush.py [hunk 2]
```diff
@@ -8,17 +8,30 @@
...
+
     with open(args.sample, "rb") as f:
         sample = Resource("sample", f.read())
-
-    task = Task({"type": "sample", "stage": "recognized", "platform": "win32"})
     task.add_resource("sample", sample)
 
+    # Add filename
+    filename = os.path.basename(args.sample)
+    task.add_payload("file_name", os.path.splitext(filename)[0])
```
Comment (icedevml) [inline drakrun/drakrun/drakpush.py:25]: :thinking:

File: drakrun/drakrun/main.py (hunk 1 above)
Comment (icedevml) [reply drakrun/drakrun/main.py:300]: Looks like this `f"file_name.{extension}"` is not correct, key names inside `payload` dict should be always constant (i.e. static strings)

File: drakrun/drakrun/drakpush.py (hunk 2 above)
Comment (icedevml) [reply drakrun/drakrun/drakpush.py:25]: doesn't match with https://github.com/CERT-Polska/drakvuf-sandbox/pull/205#pullrequestreview-466635328

File: drakrun/drakrun/main.py (hunk 1 above)
Comment (kscieslinski) [reply drakrun/drakrun/main.py:300]: This is ofc. totally wrong. My bad. It was supposed to be a quick fix:/

**Comment by kscieslinski** [comment]
> In the frontend we have feature to manually specify the file name. Previously it applied to the full file name (i.e. file name WITH extension) so I suppose this is broken right now.
> 
> I think user should still be able to override the full file name (including extension).

Hmm, no, this should be fine. We ask user only for filename but later on in `drakcore/app.py upload endpoint`/`drakrun/drakpush.py` we divide the filename to `filename without extension` and `extension`. This is because `drakrun` requires so (for routing).

**Pull request merged**